src/background.js

```diff
@@ -1,10 +1,27 @@
-let tabHistory = {};
+/*
+ * windows object
+ *     {windowId}
+ *         currentTabId
+ *         tabHistory
+ *             "-1"
+ *                 tab object (dummy start)
+ *             {tabId}
+ *                 tab object
+ *
+ * tab object
+ *     id
+ *     prev
+ *     next
+ */
+let windows = {};
+let currentWindowId;
 let currentTabId;
 let currentTabJustRemoved = false; // No-use if `defaultTabClosingBehavior`.
 let defaultTabClosingBehavior = true;
 
+// TODO: chrome.runtime.onInstalled.addListener(function callback)
```

Comment:
@frankyjuang what do you require here?

Reply:
Oh I forgot to clean that.
I was wondering if we notify the users  about this new feature. Otherwise, they won't know the there's an update and now can use multiple windows.
`chrome.runtime.onInstalled.addListener(function callback)` is fired when extension is installed or update.

Reply:
@frankyjuang 

> I was wondering if we notify the users about this new feature

Very nice idea! Didn't that that was possible. Can you make a proposal for the notification?